src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -18,19 +22,91 @@ export interface IMutationMapperProps {
     oncoKbEvidenceCache?:OncoKbEvidenceCache;
     cancerTypeCache?:CancerTypeCache;
     mutationCountCache?:MutationCountCache;
+    pdbHeaderCache?: PdbHeaderCache;
     pmidCache?:PmidCache;
 }
 
 @observer
 export default class MutationMapper extends React.Component<IMutationMapperProps, {}>
 {
+    @observable protected is3dPanelOpen = false;
+
     constructor(props: IMutationMapperProps) {
         super(props);
+
+        this.open3dPanel = this.open3dPanel.bind(this);
+        this.close3dPanel = this.close3dPanel.bind(this);
     }
 
     public render() {
         return (
             <div>
+                {
+                    (this.is3dPanelOpen) && (
+                        // TODO this is a static instance for temporary testing purposes only.
+                        // pdbId, chainId and residues should not be static...
+                        <StructureViewerPanel
+                            pdbId="3pxe"
+                            chainId="B"
+                            pdbHeaderCache={this.props.pdbHeaderCache}
+                            onClose={this.close3dPanel}
+                            residues={[
```

Comment:
These are static test values, and will be replaced with actual data in the next PR.